.github/workflows/appsec.yml

```diff
@@ -7,6 +7,9 @@ on:
   schedule:
     - cron: '0 4 * * *'
 
+env:
+  OLDEST_NODE: 18
```

Comment:
this shouldn't be merged to v4 no ?

Reply:
I mean dont we support node 16 in v4 ?

Reply:
This just replaces some existing hard-coded instances of `18` in this file. Presumably we'd need to backport this to the v4 release branch separately, with `16` here, but I'm not sure what's currently on that branch. I haven't looked.